packages/ketchup/src/components/kup-input-panel/kup-input-panel.tsx

```diff
@@ -594,14 +597,19 @@ export class KupInputPanel {
         );
     }
 
-    #renderDataTable(cell: KupDataCell, cellId: string) {
+    #renderDataTable(
+        cell: KupDataCell,
+        cellId: string,
+        isAbsoluteLayout: boolean = false
+    ) {
         return (
             <kup-data-table
                 id={cellId}
                 editableData={true}
                 showGroups={true}
                 showFilters={true}
                 showFooter={true}
+                legacyLook={isAbsoluteLayout}
```

Comment:
Hmm.. Where is the management of this attribute?

Reply:
LegacyLook its false by default and it will be set true if the layout of the InputPanel is absolute.

Reply:
Ok.. I found out that this kup-data-table attribute is used in f-cell. That's not the right way.. but it's not your fault or your PR's fault. I'm approving your PR, but I would like to talk about it later.